File: Program_Files/ScopeForLoop.cs
/* Online C# Compiler and Editor */
using System;
class ScopeForLoop {
    static void Main() {
        for (int i = 0; i < 5; i++) {
            Console.WriteLine("Inside for loop: i = " + i);
        }
        // i is not visible outside the for loop
        // This will break the program: "error: The name 'i' does not exist in the current context"
        // Console.WriteLine("Outside for loop: i = " + i);
    }
}

File: Program_Files/ScopeForLoop.java
/* Online Java Compiler and Editor */
public class ScopeForLoop {
    public static void main(String[] args) {
        for (int i = 0; i < 5; i++) {
            System.out.println("Inside for loop: i = " + i);
        }
        // i is not visible outside the for loop
        // This will break the program: "error: cannot find symbol"
        // System.out.println("Outside for loop: i = " + i);
    }
}
